src/altinn-app-frontend/src/store/index.ts

```diff
@@ -22,7 +22,7 @@ export const setupStore = (preloadedState?: PreloadedState<RootState>) => {
 
   const innerStore = configureStore({
     reducer: reducers,
-    devTools: isDev,
+    devTools: true,
```

Comment:
Just a note: this [defaults](https://redux-toolkit.js.org/api/configureStore#devtools) to `true`, so strictly not needed config. But better to be explicit I guess! 

Neat 🥇

Reply:
Oh, didn't know that! But yeah, I don't mind that being explicit.